### DISCUSSION TITLE
Is there a way to test SQL statements when creating a view?

### DISCUSSION THREAD

**Discussion by jonshipman** [pocketbase/pocketbase#3999] [Q&A]
I'm a klutz when writing SQL but can manage to piece something together after a few times. The best method I've been using has been taking down the database and running it in sqlite3 until I get it right, then restarting up pocketbase to paste it in. Is there a way to test sql without needing to take down pocketbase, preferably in the admin ui that I'm missing?

**Comment by ganigeorgiev** [comment]
> Is there a way to test SQL statements when creating a view?

Saving the view and exploring the results in the Admin UI should work for most of the cases.
Note that view collections are read-only and admin protected by default and you don't have to restart the PocketBase process.
